CPAC/pipeline/engine.py

```diff
@@ -1949,9 +1949,6 @@ def initiate_rpool(wf, cfg, data_paths=None, part_id=None):
                 ingress_raw_func_data(wf, rpool, cfg, data_paths, unique_id,
                                       part_id, ses_id)
 
-    # grab already-processed data from the output directory
-    rpool = ingress_output_dir(cfg, rpool, unique_id, creds_path)
-
```

Comment:
:raised_eyebrow: We don't want outputs from previous runs?

Reply:
What do you mean?

Reply:
I guess this one is a question for @sgiavasis 

In `develop` we had
https://github.com/FCP-INDI/C-PAC/blob/a4ee983c5bd5e8c2adabc54daefff5871a455099/CPAC/pipeline/engine.py#L1942-L1954
, but in this branch we have https://github.com/FCP-INDI/C-PAC/blob/6dc4a2b904529504a21a91252d4dbfa3b4418fc8/CPAC/pipeline/engine.py#L1944-L1953 (removing the lines this comment thread is on https://github.com/FCP-INDI/C-PAC/blob/a4ee983c5bd5e8c2adabc54daefff5871a455099/CPAC/pipeline/engine.py#L1950-L1951)

The change comes from this commit: https://github.com/diegoaper/C-PAC_MDMR-Group_Runner/commit/94d80829eee156ead693da1bbacb1f7e2dca3fd0 ; I'm just asking if this change is intended

Reply:
I wonder if this change got pulled in from elsewhere by accident, because _I_ wanted to remove that functionality for now (it has no real effect on the pipeline but was a stub for future expansion of this feature- plus fmriprep output ingress is far too not-seamless (seamful?) for it to be useful right now). And I already made a similar change in another branch, but based off `develop`.

Either way, it can stay - some of my current `develop`-based branches are doing the same.